website/docs/topics/the-anatomy-of-unleash.mdx

```diff
@@ -0,0 +1,195 @@
+
+[**Feature toggles**](../reference/feature-toggles.mdx) are at the heart of Unleash’s functionality. Feature toggles belong to projects and live next to project environments. In and of itself, a feature toggle doesn’t do anything. You must assign [**activation strategies**](../user_guide/activation-strategies.md) to it for it to start taking effect.
+
+When creating a feature toggle, you must assign a unique (across your Unleash instance) name, a [feature toggle type](../advanced/feature-toggle-types.md), a [project](../user_guide/projects.md) it belongs to, and an optional description. Everything except for the name can be changed later.
+
+<Figure img="/img/anatomy-of-unleash-features.png" caption="Feature toggle states are evaluated independently in each environment." alt="A hierarchy showing a project containing an environment containing a feature toggle configuration."/>
+
+## Activation strategies
+
+<Figure img="/img/anatomy-of-unleash-strategy.png" caption="Activation strategies are applied to feature toggles on a per-environment basis and decide whether a feature is enabled or not." alt="A hierarchy displaying an environment containing a feature toggle configuration with an activation strategy."/>
+
+[**Activation strategies**](../user_guide/activation-strategies.md) (or just **strategies** for short) are the part of feature toggles that tell Unleash **who should get a feature**. An activation strategy is assigned to **one **feature toggle in **one **environment.
+
+When you check a [feature toggle](../reference/feature-toggles.mdx) in an application, the toggle’s strategies decide the result. As long as **at least one** of a toggle’s strategies resolve to true for the current context (user or application), then the toggle will be considered enabled. In other words, if you have a hundred strategies and ninety-nine of them resolve to false, but one of them resolves to true, then the toggle is enabled.
+
+Activation strategies tie feature toggles and [environments](../user_guide/environments.md) together. When you assign an activation strategy to a feature toggle, you do so in one environment at a time. You can assign the same strategy to the same toggle in different environments, but they will be different instances of the same strategy, and do not stay in sync. Unleash also lets you copy strategies from one environment to another.
+
+Unleash comes with a number of strategies built in (refer the [activation strategies documentation](../user_guide/activation-strategies.md) for more information on those). You can also create your own [custom activation strategies](../advanced/custom-activation-strategy.md) if you need them. All strategies can be further augmented by [**strategy constraints**](../advanced/strategy-constraints.md).
+
+
+<Figure img="/img/anatomy-of-unleash-environments-strategies.png" caption="Feature toggles exist across environments and can have different activation strategies in each environment."/>
```

Comment:
This figure and the next one include the "Segments" concept in addition to constraints. The constraints are already mentioned in this context, however the Segments will be introduced two sections down this point. I suggest we simplify this diagram and the next one by leaving just the Constraint boxes and removing the Segment boxes. There are other diagrams below where Segments are showing clearly.

Reply:
@thomasheartman @dsusa72 when we did the initial sketches weren't we going in the direction that the sequence of the images should be incrementing (with each image adding more thing into the scene?)
- now they are a bit mixed

Reply:
@NicolaeUnleash Yeah, that may be possible. I switched them around a bit because I thought they covered the various topics better in this order and also to avoid having too many images clustered together. We can change it if we want to, but I think the current order illustrates well how they work.

Reply:
I'm calling it for now: I'm gonna publish this version, and then we can make another pass at the images or their placement later. I think there's more value in getting it out there now.